test/specs/redos/cubic_def.cjs

```diff
@@ -1,4 +1,4 @@
 module.exports = {
   markdown: `[x]:${' '.repeat(1500)}x ${' '.repeat(1500)} x`,
-  html: `<p>[x]:${' '.repeat(1500)}x ${' '.repeat(1500)} x</p>`,
+  html: `<p>[x]:${' '.repeat(1500)}x ${' '.repeat(1500)} x</p>`
```

Comment:
Personally, I like trailing commas

Reply:
Me too. I was just kept the rules the same as before.

I will try changing the rules in a follow up pr. There will be a lot of src code changes